apps/src/templates/ArrowButtons.jsx

```diff
@@ -56,7 +56,7 @@ class ArrowButtons extends React.Component {
 
 const styles = {
   hidden: {display: 'none'},
-  visible: {display: 'inline-block'}
+  visible: {display: 'inline-block', marginRight: '10px'}
```

Comment:
does this fix work in RTL languages?

Reply:
Good call. I will check first thing in the morning.